ubuntu_image/builder.py

```diff
@@ -87,6 +87,10 @@ def __init__(self, args):
         self.gadget = None
         self.args = args
         self.unpackdir = None
+        self.cloud_init = (
+            None
+            if args.cloud_init is None
+            else args.cloud_init)
```

Comment:
I'm pretty sure this can just be `self.cloud_init = args.cloud_init`.  The default will be None if the cli switch isn't given.